app/routes/_gcn.news.email.tsx

```diff
@@ -21,14 +22,17 @@ import { useState } from 'react'
 
 import { getUser } from './_gcn._auth/user.server'
 import { moderatorGroup } from './_gcn.circulars/circulars.server'
-import { sendAnnouncementEmail } from './_gcn.user.email/email_announcements.server'
+import {
+  announcementAppendedText,
+  sendAnnouncementEmail,
+} from './_gcn.user.email/email_announcements.server'
 import { getFormDataString } from '~/lib/utils'
 
 export async function loader({ request }: LoaderFunctionArgs) {
   const user = await getUser(request)
   if (!user?.groups.includes(moderatorGroup))
     throw new Response(null, { status: 403 })
-  return null
+  return announcementAppendedText
```

Comment:
Rather than pass this through the loader, just import it. Move the definition of this symbol to a file whose extension is not `.server.ts`.